packages/core/src/makeStyles.ts

```diff
@@ -12,6 +12,11 @@ export function makeStyles<Slots extends string | number>(stylesBySlots: StylesB
   let ltrClassNamesForSlots: Record<Slots, string> | null = null;
   let rtlClassNamesForSlots: Record<Slots, string> | null = null;
 
+  let sourceURL: string | undefined;
+  if (process.env.NODE_ENV !== 'production' && isDevToolsEnabled) {
+    sourceURL = getSourceURLfromError();
+  }
```

Comment:
Should not be the same done for `__styles`?

Reply:
it is in __styles :)